src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -60,22 +59,44 @@ public ApiController (String apiPrefix, TypedPersistence<T> persistence, String
     }
 
     /**
-     * This method is called by {@link SparkSwagger} to register endpoints and generate the docs.
+     * This method is called on each object deriving from ApiController by {@link SparkSwagger}
+     * to register endpoints and generate the docs.
+     * In this method, we add the different API paths and methods (e.g. the CRUD methods)
+     * to the restApi parameter for the applicable applicable controller.
      * @param restApi The object to which to attach the documentation.
      */
     @Override
     public void bind(final SparkSwagger restApi) {
-        LOG.info("Registering routes and enabling docs for {}", ROOT_ROUTE);
+        ApiEndpoint apiEndPoint = restApi.endpoint(endpointPath(ROOT_ROUTE)
+            .withDescription("Interface for querying and managing '" + classToLowercase + "' entities."),
```

Comment:
Can this be formatted better so that the function arguments have the same offset from the start of the line?